pkg/controller/apischeme/apischeme_controller.go

```diff
@@ -72,13 +75,14 @@ type ReconcileAPIScheme struct {
 	scheme *runtime.Scheme
 }
 
-// Reconcile reads that state of the cluster for a APIScheme object and makes changes based on the state read
-// and what is in the APIScheme.Spec
-// TODO(user): Modify this Reconcile function to implement your Controller logic.  This example creates
-// a Pod as an example
-// Note:
-// The Controller will requeue the Request to be processed again if the returned error is non-nil or
-// Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
+// Reconcile will ensure that the rh-api management api endpoint is created and ready.
+// Rough Steps:
+// 1. Create AWS ELB (CreatingLoadBalancer)
+// 2. Create Security Group with allowed CIDR blocks (UpdatingCIDRAllownaces)
+// 3. Add master Node EC2 instances to the load balancer as listeners (6443/TCP) (UpdatingLoadBalancerListeners)
+// 4. Update APIServer object to add a record for the rh-api endpoint (UpdatingAPIServer)
```

Comment:
step 4 is DNS, right?

Reply:
1. Create the ELB (add listeners)
2. Ensure CIDR access (create SG, attach SG to ELB)
3. Add instances to ELB
4. Ensure DNS
5. Ensure APIServer/cluster object is updated